### DISCUSSION TITLE
Add functionality to draw tile overlays under objects/players/npcs

### DISCUSSION THREAD

**Discussion by alexanderhenne** [runelite/runelite#14967] [Ideas]
This would make it so that tile border overlays are rendered before entities, effectively making it so they do not obstruct the entities like they do currently.

**Before:**

![](https://runelite.net/img/features/mlm.png)

**After:**

![](https://i.imgur.com/jCncVCq.png)

(excuse the quickly done image editing job)

This would provide a more organic look for these types of overlays I think.

**NOTE:** This would sadly not enable drawing a whole overlay under objects/players/npcs, as the fact that the game has bridges makes that practically impossible.

**Comment by 3Ustin** [comment]
I was going to write a issue for this, but saw that it was turned into a discussion. This would be a huge win to bridge the gap between my hardcore friend and my casual chill self. Thanks.
